kubectl_plugins/create-service/create-service.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/kubernetes-incubator/service-catalog/kubectl_plugins/utils"
+	v1alpha1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1alpha1"
+	clientset "github.com/kubernetes-incubator/service-catalog/pkg/client/clientset_generated/clientset"
+
+	"k8s.io/client-go/rest"
+)
+
+const USAGE = `
+
+Usage:
+  kubectl plugin create-service SERVICE_CLASS_NAME PLAN_NAME NAMESPACE/INSTANCE_NAME
```

Comment:
I am going with `create-service-instance` for now.

See my comment above regarding `--namespace`